### DISCUSSION TITLE
Release new version for sphinx 8 support

### DISCUSSION THREAD

**Discussion by kasium** [pradyunsg/furo#824] [Questions] [answered]
Hi folks,

it would be great to get a new furo version to support sphinx 8. Any plans yet?

Thanks!

**Comment by pradyunsg** [comment, marked as answer]
Done! https://pradyunsg.me/furo/changelog/#energetic-eminence

**Reply by kasium** [reply]
Thanks a lot!

**Reply by edgarrmondragon** [reply]
~~Unfortunately the release workflow failed 😞~~

https://github.com/pradyunsg/furo/actions/runs/10262724329/job/28393022203

**Reply by JohnVillalovos** [reply]
It was published though: https://pypi.org/project/furo/2024.8.6/

**Reply by pradyunsg** [reply]
> [pradyunsg/furo/actions/runs/10262724329/job/28393022203](https://github.com/pradyunsg/furo/actions/runs/10262724329/job/28393022203)

Yea, that one goes to TestPyPI and is something I'm slowly iterating on with each release. :)
